src/main/resources/config/liquibase/changelog/20250911170100_changelog.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <changeSet id="20250819203300-factor-out-calendar-subscription-token-into-store" author="marlonnienaber">
+        <!-- Only execute this changeset if the old calendar_subscription_token column exists -->
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="jhi_user" columnName="calendar_subscription_token"/>
+        </preConditions>
+
+        <!-- Create new token store table with token property. Connect user table via 1-to-1 relationship -->
+        <createTable tableName="calendar_subscription_token_store">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="token" type="varchar(32)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="jhi_user_id" type="bigint">
+                <constraints nullable="false" unique="true"/>
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint
+                baseColumnNames="jhi_user_id"
+                baseTableName="calendar_subscription_token_store"
+                constraintName="FK_CALENDAR_SUBSCRIPTION_TOKEN_STORE_ON_USER"
+                deferrable="false"
+                initiallyDeferred="false"
+                onDelete="CASCADE"
+                onUpdate="CASCADE"
+                referencedColumnNames="id"
+                referencedTableName="jhi_user"
+                validate="true"/>
+
+        <!-- Transfer tokens into the new table -->
+        <sql>
+            INSERT INTO calendar_subscription_token_store (token, jhi_user_id)
+            SELECT jhiUser.calendar_subscription_token, jhiUser.id
+            FROM jhi_user jhiUser
+            WHERE jhiUser.calendar_subscription_token IS NOT NULL;
+        </sql>
```

Comment:
not needed for prod environments, let's keep the prod environments clean from test server issues